certora/spec/timelockAuthorizerAlex.spec

```diff
@@ -179,12 +180,12 @@ rule onlyPendingRootCanBecomeNewRoot(method f, env e){
 
     address root_ = _root();
     
-    assert root_ == _root,
+    assert root_ == _root,    // this is verified ?  can't root change? 
```

Comment:
no, next line is a real assert

Reply:
The commented out line should be the assert instead:
`assert root_ == _root || root_ == _pendingRoot`